docs/get_started/getting_started_riscv_cmake.md

```diff
@@ -35,21 +29,37 @@ $ ./build_tools/riscv/riscv_bootstrap.sh
 
 ## Configure and build
 
-### Configure on Linux
+### Host configuration
+
+Build and install at least the compiler tools on your host machine, or install them from a binary distribution:
+
+```shell
+$ mkdir build-host ; cd build-host ; mkdir install
+$ cmake -G Ninja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=./install ..
+$ cmake --build . --target install
+```
+
+Debugging note: if `IREE_LLVMAOT_LINKER_PATH` is set for targeting Android then the build above will fail, and you should run `unset IREE_LLVMAOT_LINKER_PATH`.
+
+### Target configuration
 
 ```shell
+# Back to iree root folder.
 $ mkdir build-riscv ; cd build-riscv
 $ cmake -G Ninja \
   -DCMAKE_TOOLCHAIN_FILE="../build_tools/cmake/riscv.toolchain.cmake" \
+  -DIREE_HOST_BINARY_ROOT=$(realpath ../build-host/install) \
   -DRISCV_CPU=rv64 \
-  -DIREE_BUILD_COMPILER=OFF -DIREE_BUILD_SAMPLES=OFF \
-  -DIREE_HOST_C_COMPILER=`which clang` -DIREE_HOST_CXX_COMPILER=`which clang++` ..
+  -DIREE_BUILD_COMPILER=OFF \
+  -DIREE_BUILD_SAMPLES=OFF \
+  -DIREE_ENABLE_MLIR=OFF \
+  ..
 ```
 
 *   The above configures IREE to cross-compile towards `rv64` cpu platform.
 *   If user specify different download path (default in `${HOME}/riscv`) in `riscv_bootscrap.sh`, please append `-DRISCV_TOOL_PATH="/path/to/the/downloaded/folder"` in cmake command.
 
-### Build all targets
+build target:
 
 ```shell
 $ cmake --build .
```

Comment:
nit: If -B is used during configuration, the build path needs to be updated too.